### DISCUSSION TITLE
【问题已解决，请教原因】域名停靠cloudflare，勾选Auto Minify中的HTML，每个页面都会重复两次

### DISCUSSION THREAD

**Discussion by lzy1960** [vuepress-theme-hope/vuepress-theme-hope#2991] [Q&A] [answered]
## 部署方式

github page自定义域名，域名停靠cloudflare

## 出现的问题

如图所示，同一个页面会渲染两次（本地没问题，使用github.io域名也没问题）

![出问题的样子](https://user-images.githubusercontent.com/40552111/230076622-8a1c108e-461f-4d3b-952a-7bb9f78a3853.png)

## 复现方式

域名的dns解析使用cloudflare，并且勾选Auto Minify中的HTML，如图

![cf设置](https://user-images.githubusercontent.com/40552111/230078356-5d7f0244-7324-439c-94d1-93bf55e01538.png)

## 解决办法

将Auto Minify的HTML不勾选，即可解决

## 疑问

试了一些其他的项目，无法复现上述问题，为什么会出现这个问题呢？

**Comment by Mister-Hope** [comment, marked as answer]
Didn't the docs tell you everything?
https://theme-hope.vuejs.press/zh/faq/common-error.html#hydration-completed-but-contains-mismatches

**Reply by lzy1960** [reply]
对不起，当时没有在站内搜到这个错误，感谢指路🤯
